packages/dds/task-manager/api-report/task-manager.api.md

```diff
@@ -37,10 +37,10 @@ export interface ITaskManagerEvents extends ISharedObjectEvents {
     (event: "lost", listener: TaskEventListener): any;
 }
 
-// @internal
+// @alpha
 export type TaskEventListener = (taskId: string) => void;
 
-// @internal @sealed
+// @alpha @sealed
 export class TaskManager extends SharedObject<ITaskManagerEvents> implements ITaskManager {
     constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes);
```

Comment:
TaskManager is the only thing directly imported from this package.

The other types from this package are transitively exposed via TaskManager's public API.